app/models/match_decisions/record_client_housed_date_housing_subsidy_administrator.rb

```diff
@@ -94,9 +94,9 @@ def completed
         @decision.next_step.initialize_decision!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.

Reply:
changed.